output/mistral_splunk/mistral_splunk.c

```diff
@@ -274,6 +284,8 @@ void mistral_startup(mistral_plugin *plugin, int argc, char *argv[])
         {"skip-ssl-validation", no_argument, NULL, 'k'},
         {"token", required_argument, NULL, 't'},
         {"var", required_argument, NULL, 'v'},
+        {"cert-dir", required_argument, NULL, 'd'},
```

Comment:
best to use the same short code in all the plugins. Otherwise we're bound to get confused somewhere down the line.